includes/class-wc-siftscience-admin.php

```diff
@@ -156,14 +160,12 @@ private function output_settings_main() {
 		}
 
 		/**
-		 * Create the HTML for the checkbox
-		 *
-		 * @param string $label_for The ID of the field.
+		 * Echoing the style rule for the next sibbling of checkbox label to display inline
 		 *
-		 * @return string The outputted HTML
+		 * @param string $label_for same of The ID of the checkbox html validation.
 		 */
 		private function styling_checkbox_label( $label_for ) {
-			return sprintf( '<style type="text/css">label[for="%1$s"]+p{display:inline}</style>', $label_for );
+			echo wp_kses( sprintf( '<style type="text/css">label[for="%1$s"]+p{display:inline}</style>', $label_for ), self::ALLOWED_HTML );
```

Comment:
This line is quite long. I think it would look better if you split it into two statements:

```
$html = sprintf(....)
echo wp_kses( $html.....)
```

Reply:
your right but come to think of it no need to 

as you like